README.md

```diff
@@ -580,6 +582,93 @@ The view function here is only an example. In practice thunks are only
 relevant if you are rendering a complicated view that takes
 significant computational time to generate.
 
+## JSX
+
+[JSX](https://facebook.github.io/jsx/) is a xml-like syntax extension to ecmascript.
+Instead of using `h(tag, props, children)` to define the virtual tree,
+snabbdom supports using jsx in babel or tsx in typescript.
```

Comment:
Same here...

Reply:
This comment feels a bit vague. What do you mean by `here` ?

I'd really appreciate more actionable PR comments in the format.

`{thing} is/has {issue} because {reason}, How about {suggestion} so it could be {ideal thing}?`

Reply:
Cases of letters are wrong IMHO. `snabbdom` -> `Snabbdom`, `babel` -> `Babel`, `jsx` -> `JSX`, `tsx` -> `TSX`, `typescript` -> `TypeScript`